staging-service/src/main/java/com/github/cafdataprocessing/services/staging/StagingController.java

```diff
@@ -94,15 +94,15 @@ public ResponseEntity<Void> createOrReplaceBatch(final String X_TENANT_ID, final
             try {
                 fileItemIterator = fileUpload.getItemIterator(request);
             } catch (final IOException ex) {
-                LOGGER.error("Error getting FileItemIterator", ex);
+                LOGGER.warn("Error getting FileItemIterator: {}", ex.getMessage());
```

Comment:
I feel like the stack trace might be useful in this case.

Reply:
Ya it makes sense.